web/src/SidebarItemView.stories.tsx

```diff
@@ -45,7 +45,9 @@ function withBuildStatusOnly(status: ResourceStatus): optionFn {
     item.buildStatus = status
     item.runtimeStatus = ResourceStatus.None
     if (status === ResourceStatus.Building) {
-      item.currentBuildStartTime = new Date(Date.now() - 1).toISOString()
+      item.currentBuildStartTime = {
+        seconds: new Date(Date.now() - 1).toISOString(),
```

Comment:
I'm surprised TS compiler is allowing this! (I think it'll actually work though since we just pass the `seconds` value through to the component which can presumably handle both formats.

FWIW `fromDate()` might be helpful...assuming it exists on `MicroTime`, if not: `Math.floor(value.getTime() / 1000)`. Source: https://github.com/protocolbuffers/protobuf/blob/436bd7880e458532901c58f4d9d1ea23fa7edd52/src/google/protobuf/compiler/js/well_known_types_embed.cc#L92-L104

Reply:
i changed my mind on this -- I'm going to try to hack MicroTime to serialize differently. hold on...

Reply:
ditto